lib/lotus/generators/action.rb

```diff
@@ -108,6 +114,14 @@ def assert_action!
         end
       end
 
+      # @since x.x.x
+      # @api private
+      def assert_http_method!
+        if !HttpRouter::Route::VALID_HTTP_VERBS.include?(_http_method.upcase)
```

Comment:
There does not seem another list of supported methods. So we:

* duplicate it in the CLI
* add methods to Lotus::Router (```Lotus::Router.supported_http_methods``` and/or ```Lotus::Router.http_method_supported?('bla')```)
* move the array to Lotus::Router
* Leave as is